src/logic.js

```diff
@@ -265,6 +264,21 @@ define([
             });
             return messages;
         },
+        addReferences: function (mug, property, value) {
+            // get absolute paths from mug property's value
+            var isLabel = false,
+                _this = this;
+            if (!value && mug.p[property] && _.isFunction(mug.p[property].get)) {
```

Comment:
is checking for a `get` function really the best/only way to determine a property is a label? seems roundabout/fragile.